package.json

```diff
@@ -71,7 +71,7 @@
 	"packageManager": "yarn@4.0.2",
 	"lint-staged": {
 		"*.{js,jsx,ts,tsx,json}": [
-			"prettier --write --cache --log-level=error"
+			"prettier --write --cache --log-level=warn"
```

Comment:
unrelated change methinks

Reply:
unrelated but needed (prettier calls file errors warnings) and i cba to split it out - mind if it keep it in?